test/integration/start_stop_delete_test.go

```diff
@@ -294,7 +294,6 @@ func testPulledImages(ctx context.Context, t *testing.T, profile string, version
 	for _, img := range jv["images"] {
 		for _, i := range img.Tags {
 			// Remove container-specific prefixes for naming consistency
-			i = strings.TrimPrefix(i, "docker.io/")
 			i = strings.TrimPrefix(i, "localhost/")
```

Comment:
probably remove this as well ? (it is what you get when loading local files with e.g. podman)

Reply:
actually I was wrong, we need to keep **both** of them. And then apply the docker.io transformation also to "want"

Reply:
done